CHANGELOG.md

```diff
@@ -7,64 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 Headers are one of:
 
-- `Added`, `Changed`, `Removed` or `Fixed`.
+-   `Added`, `Changed`, `Removed` or `Fixed`.
```

Comment:
Seems like this is not necessary

Reply:
The space, or the whole line? (It came from the forked code)

Reply:
The space

Reply:
Prettier did that.